docs/modules/guides/pages/use-in-memory-db-to-manage-test-data.adoc

```diff
@@ -0,0 +1,43 @@
+= How to use in-memory database to manage test data
+
+Assuming there is a project containing VIVIDUS-based tests
+(e.g. https://github.com/vividus-framework/vividus-starter[VIVIDUS Starter])
+and the goal is to manage test data in the in-memory database (e.g.
+https://h2database.com/html/features.html#feature_list[H2 database]).
+The database is deployed on tests start, accessible from tests and shutdown on
+tests end. The test data can be created from SQL files located in the project.
+
+. Add the required dependencies (DB plugin, H2 database) to the project `build.gradle`
++
+[source,gradle,subs="attributes+"]
+----
+implementation(group: 'org.vividus', name: 'vividus-plugin-db', version: '{current-version}')
+implementation(group: 'com.h2database', name: 'h2', version: '1.4.200')
+----
+
+. Configure connection to the H2 database
++
+[source,properties]
+----
+db.connection.testh2db.driver-class-name=org.h2.Driver
+db.connection.testh2db.url=jdbc:h2:mem:test;\
+  INIT=create schema if not exists test\\;\
+  SET SCHEMA test\\;\
+  RUNSCRIPT FROM 'classpath:/path/to/sql/file1.sql'\\;\
```

Comment:
I think it makes sense to describe what is classpath:/ prefix is and how it maps to the regular VIVIDUS test project